internal/output/diff.go

```diff
@@ -294,9 +294,5 @@ func getSym(d decimal.Decimal) string {
 		return "+"
 	}
 
-	if d.IsNegative() {
```

Comment:
@ekifel I'm not sure this is the correct fix. Looking at your example the Cost change should also be showing a negative not a +.

Reply:
Woops, I did not notice that

Reply:
@aliscott, fixed. I updated PR comment with all of cases